source/crud/tests/v2/find-allowdiskuse.yml

```diff
@@ -0,0 +1,50 @@
+runOn:
+  - { minServerVersion: "4.4.0" }
```

Comment:
[SERVER-42077](https://jira.mongodb.org/browse/SERVER-42077) was released in 4.3.1, so we should use that version here. That will also make it easier for drivers to run these tests without any manual modifications (or worse, waiting until 4.4.0 is actually released). This would be consistent with many other recent spec tests.

Reply:
Done